### ISSUE TITLE
[modals] flayout variant doesn't like selects

### ISSUE THREAD

**Issue by helderneves91** [livewire/flux#221]
Hi,

Consider the following code, which won't show any select option:
```
<div wire:poll.10s>

    <flux:modal.trigger name="edit-profile">
        <flux:button>Edit profile</flux:button>
    </flux:modal.trigger>

    <flux:modal name="edit-profile" variant="flyout" class="space-y-6">

        <flux:select wire:model="industry" placeholder="Choose industry...">
            <flux:option>Photography</flux:option>
            <flux:option>Design services</flux:option>
            <flux:option>Web development</flux:option>
            <flux:option>Accounting</flux:option>
            <flux:option>Legal services</flux:option>
            <flux:option>Consulting</flux:option>
            <flux:option>Other</flux:option>
        </flux:select>

    </flux:modal>

</div>
```

<img width="1356" alt="Captura de ecrã 2024-10-01, às 17 46 50" src="https://github.com/user-attachments/assets/5aff79a0-de16-41ba-8474-de7b3e2562d4">

**Comment by patrykszady** [comment]
Are you using 1.0.6? Haven’t tested yet but was supposed to be fixed in 1.0.5. See pro changelog

**Comment by patrykszady** [comment]
Confirmed this is fixed in 1.0.5. https://github.com/livewire/flux/issues/89 https://changelog.anystack.sh/flux-pro

**Comment by helderneves91** [comment]
Hi @patrykszady , thanks. I had old flux components published. Just needed to delete the folder
